producers/src/end-to-end-test/java/uk/gov/moj/sdt/producers/SdtRunTimeException.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.moj.sdt.producers;
+
+public class SdtRunTimeException extends RuntimeException {
+    /**
+     * The Constant serialVersionUID.
+     */
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Error code.
+     */
+    // CHECKSTYLE:OFF
+
```

Comment:
Do we need the CHECKSTYLE OFF/ON notations here?

Remove blank line between JavaDoc comment and field.